opencensus/stats/aggregation.py

```diff
@@ -0,0 +1,41 @@
+from opencensus.stats import bucket_boundaries
+
+class BaseAggregation(object):
+    def __init__(self, aggregation=None):
+        self.aggregation = dict(aggregation or {})
+
+    def get_aggregation(self):
+        return self.aggregation
+
+class SumAggregation(BaseAggregation):
```

Comment:
Is the sum being calculated? And same for count, mean below.

Reply:
need to work on that..